src/main/java/ab/utils/IModHelper.java

```diff
@@ -0,0 +1,10 @@
+package ab.utils;
+
+public interface IModHelper {
+
+    public void preInit();
+
+    // public void init();
```

Comment:
It's probably cleaner to actually add this to the interface, delegate accordingly and leave it empty in the implementing classes where needed.